pkg/config/config.go

```diff
@@ -45,6 +45,23 @@ type OIDC struct {
 	FriendlyNameClaimName  string            `mapstructure:"friendly_name_claim_name"`
 }
 
+// TODO(isan) consolidate with OIDC
+// CookieAuthVerification is related to auth based on a cookie set by an external service
+type CookieAuthVerification struct {
```

Comment:
Is this a multi-purpose cookie or a SAML one? If it's multi-purpose let's change the API wording from SAML, if it's not, let's change the name here to include SAML.

Reply:
The struct itself can be multi-purpose, it's not specific to SAML. 
The API though uses this for specifically doing SAML verification.